src/bundles/APIBundle/Resources/config/services.yml

```diff
@@ -22,7 +22,8 @@ services:
 
   AlmaviaCX\Calameo\API\Serializer:
     arguments:
-      $baseSerializer: "@jms_serializer"
+      # TODO   The service "AlmaviaCX\Calameo\API\Serializer" has a dependency on a non-existent service "JMS\Serializer\SerializerInterface".
+      $baseSerializer: '@JMS\Serializer\SerializerInterface'
```

Comment:
TODO

The service "AlmaviaCX\Calameo\API\Serializer" has a dependency on a non-existent service "JMS\Serializer\SerializerInterface".